### PR TITLE
fix: import pl into routes & callback

### PR DESCRIPTION
**Pull request by danielgent**
and tested :heavy_check_mark:

### CODE REVIEW

File: src/routes/tsx/routes.tsx [hunk 1]
```diff
@@ -93,6 +94,7 @@ function initI18n(lng: string) {
       it: { translation: it },
       nb: { translation: nb },
       sv: { translation: sv },
+      pl: { translation: pl },
```
Comment (danielgent) [inline src/routes/tsx/routes.tsx:97]: I would KISS and extract out `initI18n` into a helper function so we only have to do this in one place, but I know you had greater dreams with your Pr with the :x:  :smile:

File: test/integration/login/code-flow-liquidjs.spec.ts [hunk 2]
```diff
@@ -245,8 +260,10 @@ describe("Login with code on liquidjs template", () => {
     const { to, code, subject } = getCodeAndTo(env.data.emails[0]);
     expect(to).toBe("bar@example.com");
     expect(subject).toBe(
-      `Välkommen till Test Tenant! ${code} är koden för att logga in`,
+      // `Välkommen till Test Tenant! ${code} är koden för att logga in`,
+      `Benvenuti su Test Tenant! ${code} è il codice di accesso`,
```
Comment (danielgent) [inline test/integration/login/code-flow-liquidjs.spec.ts:264]: :thinking: That's not Polish! :smile:

File: src/utils/i18n.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import en from "../locales/en/default.json";
 import sv from "../locales/sv/default.json";
 import nb from "../locales/nb/default.json";
 import it from "../locales/it/default.json";
-import pl from "../locales/it/default.json";
+import pl from "../locales/pl/default.json";
```
Comment (danielgent) [inline src/utils/i18n.ts:5]: Cool easter egg :saluting_face: 

ChatGpt?

Comment (danielgent) [reply src/utils/i18n.ts:5]: Same comment above where this code is duplicated twice - on `src/controllers/email.ts` and `src/utils/i18n.ts`
and I would extract it out in one place

_now_ if we've actually shut down login2 we could move all those translations into auth2...

File: src/routes/oauth2/callback.ts [hunk 4]
```diff
@@ -19,6 +20,7 @@ function initI18n(lng: string) {
       it: { translation: it },
       nb: { translation: nb },
       sv: { translation: sv },
+      pl: { translation: pl },
```
Comment (markusahlstrand) [inline src/routes/oauth2/callback.ts:23]: Ahh.. this was part of the other PR that i didn't merge..

File: src/utils/i18n.ts (hunk 3 above)
Comment (markusahlstrand) [reply src/utils/i18n.ts:5]: Ooops.. copy paste isn't always the best solution.

**Review by markusahlstrand — APPROVED** [review]
Looks good!

**Pull request merged**